src/pages/docs/surveys.tsx

```diff
@@ -104,6 +104,68 @@ export const Content = ({ quickLinks = false }) => {
                     />
                 </ul>
             </section>
+            <section>
+                <h3 className="mb-1 text-xl">Survey templates</h3>
+                <p className="text-[15px]">Choose from many common survey types and launch them in a few clicks!</p>
+
+                <ul className="m-0 mb-3 p-0 flex flex-col gap-4 md:grid md:grid-cols-2 xl:grid-cols-3">
+                    <ResourceItem
+                        title="Net Promoter Score (NPS)"
+                        description="The benchmark for judging what users think of you"
+                        Image={
+                            <StaticImage
+                                alt=""
+                                className="h-full"
+                                placeholder="none"
+                                objectFit="contain"
+                                quality={100}
+                                src="./images/nps-survey.png"
+                            />
+                        }
+                        url="/templates/nps-survey"
+                    />
+                    <ResourceItem
+                        title="Customer Satisfaction (CSAT)"
+                        description="Get a vibe check on your product"
+                        Image={
+                            <StaticImage
+                                alt=""
+                                className="h-full"
+                                placeholder="none"
+                                objectFit="contain"
+                                quality={100}
+                                src="./images/csat-survey.png"
+                            />
+                        }
+                        url="/templates/csat-survey"
+                    />
+                    <ResourceItem
+                        title="Customer Churn Rate (CCR)"
+                        description="Find out why your users are saying goodbye"
+                        Image={
+                            <StaticImage
+                                alt=""
+                                className="h-full"
+                                placeholder="none"
+                                objectFit="contain"
+                                quality={100}
+                                src="./images/ccr-survey.png"
+                            />
+                        }
+                        url="/templates/churn-survey"
+                    />
+                </ul>
+                <CallToAction
+                    to="/templates"
```

Comment:
```suggestion
                    to="/templates?filter=type&value=survey"
```

Reply:
In the [survey creation steps docs](https://posthog-git-surveys-docs-post-hog.vercel.app/docs/surveys/creating-surveys#steps), we show a list of templates in a table. The table is outdated though and missing a few templates.

We could either:
1. Update the table
2. Remove the table, and instead link to `/templates?filter=type&value=survey`.

I prefer option 2. Would you mind doing it while you're updating this docs?

Reply:
I'll update the table instead. I think there's a slightly different purpose here: the steps show all the types of surveys that are possible, whereas templates show specific implementations of those steps. They're similar, but not the same. I will add the survey template list to the bottom though too!